open_diloco/utils.py

```diff
@@ -108,10 +108,18 @@ def get_compression_kwargs(hivemind_compression: str) -> dict:
 
         ret_kwargs["grad_compression"] = NoCompression()
         ret_kwargs["state_averaging_compression"] = NoCompression()
+    elif hivemind_compression == "uniform8bit":
+        from hivemind import Uniform8BitQuantization
+
+        ret_kwargs["grad_compression"] = Uniform8BitQuantization()
+        ret_kwargs["state_averaging_compression"] = Uniform8BitQuantization()
+    elif hivemind_compression == "quantile8bit":
+        from hivemind import Quantile8BitQuantization
+
+        ret_kwargs["grad_compression"] = Quantile8BitQuantization()
+        ret_kwargs["state_averaging_compression"] = Quantile8BitQuantization()
     else:
-        raise ValueError(
-            f"Invalid hivemind_compression: {hivemind_compression}. Please choose 'none', 'fp16', or 'scaled-fp16'."
-        )
```

Comment:
the config is already catching mis naming and propose suggestion thanks to literal support in pydantic

Reply:
ah ok great!